### PR TITLE
Update enchants.json (Mana Steal, Smarty Pants, and Dragon Tracer) fixed formatting

### PR DESCRIPTION
**Pull request by Cobble8**

### CODE REVIEW

File: skyblock/enchants.json [hunk 1]
```diff
@@ -246,6 +246,12 @@
       "goodLevel": 5,
       "maxLevel": 6
     },
+    "mana steal":{
+       "loreName": "Mana Steal",
+       "nbtName": "mana_steal",
+      "goodLevel": 0,
+      "maxLevel": 3
+    },
```
Comment (ImRodry) [inline skyblock/enchants.json:254]: Formatting please

File: skyblock/enchants.json [hunk 2]
```diff
@@ -246,6 +246,12 @@
       "goodLevel": 5,
       "maxLevel": 6
     },
+    "mana steal":{
+       "loreName": "Mana Steal",
+       "nbtName": "mana_steal",
+      "goodLevel": 0,
```
Comment (ImRodry) [inline skyblock/enchants.json:252]: How is the good level 0?

File: skyblock/enchants.json [hunk 3]
```diff
@@ -336,6 +342,12 @@
       "goodLevel": 1,
       "maxLevel": 1
     },
+    "smarty pants":{
```
Comment (ImRodry) [inline skyblock/enchants.json:345]: formatting

File: skyblock/enchants.json [hunk 4]
```diff
@@ -336,6 +342,12 @@
...
+       "loreName": "Smarty Pants",
+       "nbtName": "smarty_pants",
+       "goodLevel": 0,
```
Comment (ImRodry) [inline skyblock/enchants.json:348]: how is the good level 0?

**Review by ImRodry — CHANGES_REQUESTED** [review]

File: skyblock/enchants.json (hunk 4 above)
Comment (Phoube) [reply skyblock/enchants.json:348]: Good level is the maximum level you can craft from an enchanting table, so until we make a better system for it, I think smarty pants, which is uncraftable, can stay at good=level 0

**Comment by Phoube** [comment]
Made some edits and merged. Thanks for the contrib! :D

**Pull request closed without merging**